server/controllers/login.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require("bcrypt");
+const Boom = require("@hapi/boom");
+const { validateSchema, signToken } = require("../utils");
+const { getUser } = require("../database");
+
+const login = async (req, res, next) => {
+  try {
+    const { email, password } = req.body;
+    let userID;
+    try {
+      await validateSchema.validateAsync({ email, password });
+    } catch (error) {
+      throw Boom.badRequest("invalid information");

```

Comment:
you can throw every errors by   " throw Boom.badRequest(error.details.map((e) => e.message).join('\n'))"
to show the validation error

Reply:
I know, I sent this on purpose for more security reasons.
the user cannot know which data was wrong, so he will have to check his data twice